network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -416,10 +421,103 @@ class Meta:
         verbose_name = "General Product Page"
 
 
-class BuyersGuidePage(FoundationMetadataPageMixin, Page):
+class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
+    """
+    Note: We'll likely be converting the "about" pages to Wagtail Pages.
+    When that happens, we should remove the RoutablePageMixin and @routes
+    """
+
     template = 'buyersguide/home.html'
     subpage_types = [SoftwareProductPage, GeneralProductPage]
 
+    @route(r'^about/$', name='how-to-use-view')
+    def about_page(self, request):
+        self.template = "about/how_to_use.html"
+        return self.serve(request)
+
+    @route(r'^about/why/$', name='about-why-view')
+    def about_why_page(self, request):
+        self.template = "about/why_we_made.html"
+        return self.serve(request)
+
+    @route(r'^about/press/$', name='press-view')
+    def about_press_page(self, request):
+        self.template = "about/press.html"
+        return self.serve(request)
+
+    @route(r'^about/contact/$', name='contact-view')
+    def about_contact_page(self, request):
+        self.template = "about/contact.html"
+        return self.serve(request)
+
+    @route(r'^about/methodology/$', name='methodology-view')
+    def about_methodology_page(self, request):
+        self.template = "about/methodology.html"
+        return self.serve(request)
+
+    @route(r'^about/meets-minimum-security-standards/$', name='min-security-view')
+    def about_mss_page(self, request):
+        self.template = "about/minimum_security.html"
+        return self.serve(request)
+
+    @route(r'^products/(?P<slug>[-\w\d]+)/$', name='product-view')
+    def product_view(self, request, slug):
+        # Find product by it's slug and redirect to the product page
+        # If no product is found, redirect to the BuyersGuide page
+        try:
+            product = ProductPage.objects.get(slug=slug, live=True)
+            redirect_url = product.url
+        except ProductPage.DoesNotExist:
+            redirect_url = self.url
```

Comment:
Should we redirect back to the BuyersGuide home page everytime a someone goes to /privcynotincluded/products/missing-product/? Or should we simply 404? A 404 is the proper thing to do in dev-terms, but redirecting to the buyers guide page would direct users back to what they're looking for. Thoughts?

Reply:
current behaviour is to 404, so let's keep that.

Reply:
kk, I changed this behaviour to use 404's. Thank you!